File: Anul 2/Semestrul 2/MPP/labs/Tema 5-proiect Server Csharp-Client Java/proiectSCcsharp/angajatModel/Spectacol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace proiectSCcsharp.domain
{
    [Serializable]
    public class Spectacol : Entity<long>
    {
        public string artistName { get; set; }
        public string date { get; set; }
        public string time { get; set; }
        public string place { get; set; }
        public int availableSeats { get; set; }
        public int soldSeats { get; set; }

        public Spectacol(long id,string ArtistName, string Date, string Time, string Place, int AvailableSeats, int SoldSeats): base(id)
        {
            artistName = ArtistName;
            date = Date;
            time = Time;
            place = Place;
            availableSeats = AvailableSeats;
            soldSeats = SoldSeats;
        }
        public override string ToString()
        {
            return artistName + " | " + date + " | " + time + " | " + place + " | " + availableSeats + " | " + soldSeats;
        }
    }
}

File: Anul 2/Semestrul 2/MPP/labs/Tema 5-proiect Server Csharp-Client Java/proiectServerClientJava/Domain/src/main/java/proiectServerClientJava/domain/Spectacol.java
package proiectServerClientJava.domain;

import java.io.Serializable;

public class Spectacol extends Entity<Long> implements Serializable {
    private String artistName;
    private String date;
    private String time;
    private String place;
    private int availableSeats;
    private int soldSeats;

    public Spectacol(Long id,String artistName, String date, String time, String place, int availableSeats, int soldSeats) {
        super(id);
        this.artistName = artistName;
        this.date = date;
        this.time = time;
        this.place = place;
        this.availableSeats = availableSeats;
        this.soldSeats = soldSeats;
    }

    public String getArtistName() {
        return artistName;
    }

    public void setArtistName(String artistName) {
        this.artistName = artistName;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
    public String getPlace() {
        return place;
    }

    public void setPlace(String place) {
        this.place = place;
    }

    public int getAvailableSeats() {
        return availableSeats;
    }

    public void setAvailableSeats(int availableSeats) {
        this.availableSeats = availableSeats;
    }

    public int getSoldSeats() {
        return soldSeats;
    }

    public void setSoldSeats(int soldSeats) {
        this.soldSeats = soldSeats;
    }
}